README.rst

```diff
@@ -1,4 +1,4 @@
-Neural Doodle
+Neural Doodle guide
```

Comment:
This is the general project readme, and not a dedicated "introductory use guide" of the project in the stricter sense. Thus having the heading without the word "guide" feels much more appropriate.

Also, please keep in mind that the line below the actual wording of the heading  should be adjusted to be the same width than the text of the heading itself in order to be consistent.